app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java

```diff
@@ -80,4 +80,40 @@ public static void onResponse(int responseCode) {
         }
     }
 
+    /**
+     * Last WindowsSetting constructor values. Values are checked for change to reduce log spam.
+     */
+    private static int lastAp, lastAnchorHorizontalPos, lastAnchorVerticalPos;
+    private static boolean lastZ1, lastZ2;
+
+    /**
+     * Injection point.
+     */
+    public static int[] getSubtitleWindowSettingsOverride(int ap, int anchorHorizontalPos, int anchorVerticalPos,
+                                                         boolean z1, boolean z2) {
+        int[] override = {ap, anchorHorizontalPos, anchorVerticalPos};
+
+        if (SettingsEnum.DEBUG.getBoolean()) {
+            if (ap != lastAp || anchorHorizontalPos != lastAnchorHorizontalPos
```

Comment:
Quite a big if block. Maybe this can be refactored into simpler code if done return oriented?